packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -28,7 +28,7 @@ export enum ButtonDirection {
 }
 
 export interface HorizontalOverflowState {
-    itemsHeight: number;
+    itemsHeight?: number;
```

Comment:
What is the reason for making `itemsHeight` an optional property? It always exists and is set. Is it more accurate to change to `itemsHeight: number | null` to make clear that `null` is a valid value of this property?

Reply:
The issue currently is that itemsHeight defaults to 0px as it is expected to be updated by componentDidMount() or by itemsLoaded() callbacks.
However these are not triggered server side so itemsHeight has an incorrect value of 0px (and technically incorrect at points on client side but a non-issue as componentDidMount/itemsLoaded will update the state)
So I was making it optional to indicate that it hadn't been set. An alternative would be to use NaN as the default instead

Reply:
It *is* always set though right? Sorry if I'm missing something, but it appears it is always set to a pixel value or `null`, meaning there is never a case where the property *doesn't exist*

Reply:
Ah new to typescript, my bad thought the `?` meant nullable as opposed to optional. Will remove the `?`.